src/camvidlog2/queries.py

```
@@ -66,3 +69,34 @@ def calculate_distances(
     return distances
+
+
+class Result(BaseModel):
+    rank: int
+    score: float
+    filename: str
+    frame_no: int
+
+
+def calculate_results(
+    distances: pd.Series, num: int = 15, roll: int = 0
```

Ruff (0.11.9)

Unused function argument: `roll`

(ARG001)